client/js/shout.js

```diff
@@ -366,7 +366,7 @@ $(function() {
 			}
 		);
 		if ([
-			"join",
+			"join","j",
```

Comment:
This is wrong. It seems you just added `"j"` wherever you saw `"join"`. In this part, it's for the settings that hide certain messages in the chat feed.